### ISSUE TITLE
will there be a larger model repository available for this framework in the future?

### ISSUE THREAD

**Issue by Leggyweggy** [intel/openvino-plugins-ai-audacity#52]
would love to see it!

**Comment by RyanMetcalfeInt8** [comment]
Hi @Leggyweggy,

Yes, we are cooking up more models / features. Anything in particular you'd like to see?

Thanks!
Ryan

**Comment by Leggyweggy** [comment]
yes actually! on top of maybe providing alternative models for noise reduction and other existing features to try and get more optimal results, it would be interesting to see an AI audio "super sampler" that tries to recover low quality or low-bit rate audio to a higher quality output. the easiest parallel I can think of that exists to this is Adobe's AI speech enhancer which definitely is good at sonically increasing audio clarity and does more than just noise reduction, but of course that's not a generalized method and only works for speech. thanks for responding! I'm loving where this project is going so far and look forward to what's to come.

**Comment by Elshara** [comment]
I would love to see a model capable of de-auto-tuning music. More specifically, a model capable of analyzing the song itself and applying auto-tune automatically based on templates. Including turning it off on songs, or applying it to vocal and or music tracks. Including custom pitch correction or scale enhancements.
Call it a silly feature, but, honestly, there are a ton of country vocals these days that amp up the style to where it's extremely robotic especially on longer vocals as the presets for whom ever got the artist to sound that way made it happen. Similarly, a ton of non English songs, use it far too broadly in my opinion.
Having said that, vocal cloning, speech enhancers, and speech to music, music to speech, or speech to speech and music to music dynamic auto adjustments to vocal sampling would be nice to have.
I'd also love the ability to generate text to speech in one of these models, or text to custom clone, where you can input a voice or vocal project, and have different voices test out different pitches.
It also would be cool to have text to instrument, instrument to instrument, and instrument to note implementations as well. Imagine adding a guitar track into audacity, analyzing it using the instrument to note feature, and it exporting sheet music for that track?
I'd love a midi synthesizer model capable of generating something compatible with sounds that you create with it, on any keyboard interface.
It'd be neat to have models for effect generation. Where you describe the effect you want, and it applies it to the track or track selection itself. For example, a reverse echo at 02:40 tapering off to an echo at 02:42 when the emphasis at peak DB range -0.9 - 0.0 Db is reached.
I'd love to be able to have a dynamic equalizer model, where you can use it to have all micro frequency ranges of the track itself as a temporal outflow appear the same or obtain a custom DB range. For example, when ever someone says the letter N, add 4 DB to 276.8 HZ. But when ever a different person says the letter S, reduce 2400 - 2950.7 HZ by 6.2 DB. At 04:50 to 04:51.
Dynamic audio generation is what I'd love to see. Imagine feeding a replicator of a track to a model, and have it dynamically recreate or edit each portion of a track itself without having any weird side effects. So you could for instance, replace silence starting at 29 DB with a horn honking at 26 DB. Where didd that come from? a file you select, or a custom generated sound.
It would also be good to have AI automatically adjust pitch, speed and tempo and isolate certain parts of songs when remixing tracks together. Or to be able to output statistics based on music tracks. Such as a BPM finder, a pitch analyzer, or a note shifter. To use micro generative audio generations where needed to have things seamlessly flow together.
I'd like to see the ability to generate video along with audio in some way as well.
These are just some ideas.
